doc/python/axes.md

```diff
@@ -35,119 +34,141 @@ jupyter:
     thumbnail: thumbnail/axes.png
 ---
 
+This tutorial explain how to set the properties of Cartesian axes, namely [`go.layout.XAxis`](/python/reference/#layout-xaxis) and [`go.layout.YAxis`](python/reference/#layout-xaxis). Other kinds of axes are described in other tutorials:
+- [3D axes](/python/3d-axes) The axis object is [`go.layout.Scene`](/python/reference/#layout-scene)
+- [Polar axes](/python/polar-chart/). The axis object is [`go.layout.Polar`](/python/reference/#layout-polar)
+- [Ternary axes](/python/ternary-plots). The axis object is [`go.layout.Ternary`](/python/reference/#layout-ternary)
+- For Geo axes, see [this choropleth example](/python/choropleth-maps/#world-choropleth-map). The axis object is [`go.layout.Geo`](/python/reference/#layout-geo)
+- For Mapbox axes, see the different mapbox tutorials in the [Maps page](/python/maps/). The axis object is [`go.layout.Mapbox`](/python/reference/#layout-mapbox)
+- Color axes are described in the [color scales tutorial](/python/colorscales/).
+
 **See also** the tutorials on [subplots](/python/subplots) and [multiple axes](/python/multiple-axes/).
 
-#### Toggling Axes Lines, Ticks, Labels, and Autorange
+The different types of Cartesian axes are 
+- 'linear'
+- 'log' (see the [example below](#logarithmic-axes))
+- 'date' (see the [tutorial on timeseries](/python/time-series/))
+- 'category' (see for example [Bar Charts](/python/bar-charts/))
+- 'multicategory' (see the [example below](#subcategory-(multicategory)-axes) ))
 
```

Comment:
I would add just one more little list saying "other than the type, here are the other things you can control at the axis level: ticks, tick labels, titles, domains, gridlines..."